ignite/cmd/plugin.go

```diff
@@ -27,18 +28,35 @@ const (
 // If no configuration found, it returns w/o error.
 func LoadPlugins(ctx context.Context, rootCmd *cobra.Command) error {
 	// NOTE(tb) Not sure if it's the right place to load this.
-	chain, err := newChainWithHomeFlags(rootCmd)
+	cfg, err := parseLocalPlugins(rootCmd)
```

Comment:
This make me realize that we can have local plugin config outside of a chain, I wonder if we should avoid that or not.

Reply:
I think it is fine for now.  There may be some use case where someone is working with `ignite` in  a specific directory  but it is not a scaffolded chain.

Reply:
OK